vic/drivers/shared_all/src/generate_default_state.c

```diff
@@ -53,15 +53,23 @@ generate_default_state(all_vars_struct *all_vars,
     };
     double                   Cv;
     double                   tmp;
+    double                   AreaFactor;
+    double                   TreeAdjustFactor = 1.;
+    double                   lakefactor = 1.;
+    double                   albedo_sum;
     double                ***tmpT;
     double                 **tmpZ;
     int                      ErrorFlag;
 
     cell_data_struct       **cell;
     energy_bal_struct      **energy;
+    veg_var_struct         **veg_var;
+    gridcell_avg_struct      gridcell_avg;
 
     cell = all_vars->cell;
     energy = all_vars->energy;
+    veg_var = all_vars->veg_var;
+    gridcell_avg = all_vars->gridcell_avg;
```

Comment:
if `gridcell_avg` is a pointer, then this would read
``` C
gridcell_avg = &(all_vars->gridcell_avg);
```
Or just get rid of it. Seems to me that later in the same code you just use `all_vars->gridcell_avg` directly anyway, so you don't really need this.